jenkins-build-pack-website/Jenkinsfile

```diff
@@ -646,7 +647,7 @@ def updateCloudFrontPolicy(String service, String s3Bucket){
 		new_policy.put('Effect',"Allow")
 		new_policy.put('Action',"s3:GetObject")
 		new_policy.put('Principal', principal)
-		new_policy.put('Resource',"arn:aws:s3:::$s3Bucket/$service/*")
+		new_policy.put('Resource',"arn:aws:s3:::$s3Bucket/*")
```

Comment:
I hope s3 bucket per service is not implemented yet in OSS.  Why the $service is removed from resource ?

Reply:
This is correct, we don't need $service in OSS.